lib/queue.js

```diff
@@ -4,6 +4,8 @@
 var redis = require('redis');
 var events = require('events');
 var util = require('util');
+var assert = require('assert');
```

Comment:
we should not use assert in production code, it is a library aimed for unit testing.